### ISSUE TITLE
import Transition: "React.createElement: type is invalid"

### ISSUE THREAD

**Issue by femans** [reactjs/react-transition-group#254]
Copied this from the example:
```
import { CSSTransition, Transition, TransitionGroup } from 'react-transition-group'
console.log("Transition:", Transition)

const transitionStyles = {
  entering: { opacity: 0 },
  entered: { opacity: 1 },
};

const Fade = ({ in: inProp }) => (
  <Transition in={inProp} timeout={300}>
    {(state) => (
      <div style={{
        // ...defaultStyle,
        ...transitionStyles[state]
      }}>
        
      </div>
    )}
  </Transition>
);

```
I installed react-transition-group version 2.2.1 with yarn. The output on the console here is

> Transition: undefined

and the Fade component throws an error: 

> React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in.

Am I overlooking something?

**Comment by koba04** [comment]
@femans Can you show your build configuration files or a repository to reproduce it?

**Comment by joris-rotteveel** [comment]
@femans , what version are you using in your package.json? I think the Transition Object is available from v2?

**Comment by femans** [comment]
Hi @koba04 and @joris-rotteveel, 
I tried to reproduce it, but could not. In the project I am using it for, I went for another solution, and got that working now, so I am not going to change that around again. 
I created a new project, copied all the configuration files of that project, added this code to it, and it works fine. So I don't know what went wrong there. I will close the issue for now. Thanks!
